### PR TITLE
Hyperlinks

### PR DESCRIPTION
**Pull request by der-lyse**
Alright, please have a look. I rebased it on top of master, squashed plenty commits and hope, I didn't miss anything. Well, this merge request for issue #10 got way bigger than I first anticipated. ¯\_(ツ)_/¯

### CODE REVIEW

**Comment by Minoru** [comment]
Thank you! Nighttime here already, will make a review in about 18 hours.

File: doc/newsboat.txt [hunk 1]
```diff
@@ -50,33 +50,33 @@ themselves depend on other libraries. These dependencies are not listed here.
...
-- GNU gettext (on systems that don't provide gettext in the libc):
-  ftp://ftp.gnu.org/gnu/gettext/
-- [pkg-config](http://pkg-config.freedesktop.org/wiki/)
-- [libxml2](http://xmlsoft.org/downloads.html)
-- [json-c (version 0.11 or newer)](https://github.com/json-c/json-c/wiki)
+- http://www.clifford.at/stfl/[STFL (version 0.21 or newer)]
+- http://www.sqlite.org/download.html[SQLite3 (version 3.5 or newer)]
+- http://curl.haxx.se/download.html[libcurl (version 7.18.0 or newer)]
+- GNU gettext (on systems that don't provide gettext in the libc:
+  ftp://ftp.gnu.org/gnu/gettext/)
```
[the remark this answers is not in the document]
Comment (der-lyse) [reply doc/newsboat.txt]: To me it looks the text "on systems that don't provide gettext in the libc" is an introduction to the link, so the whole thing should be placed in parenthesis. However, I have to say that I struggled quite a bit with this bullet point and first I wanted to get rid of the explanation completely and just link the text like all the other dependencies. But then I thought, maybe this information might be quite important to somebody, so I kept it and just "fixed" the placement of the closing parenthesis.

I hope this makes sense. Please correct me if I'm wrong.

File: doc/newsboat.txt [hunk 2]
```diff
@@ -505,20 +500,20 @@ are added to the query feed.
...
-feed>" specifies the name under which the query feed shall be displayed in the
-feed list, and "<filter expression>" is the filter expression that shall be
+The `query:` in the beginning tells newsboat that it's a query feed, `<name of
+feed>` specifies the name under which the query feed shall be displayed in the
+feed list, and `<filter expression>` is the filter expression that shall be
 used. Like every other feed, a query feed can be tagged to organize it like
 a regular feed.
 
-A good example for the user of this feature is a query feed that contains all
+A good example for the use of this feature is a query feed that contains all
```
[the remark this answers is not in the document]
Comment (der-lyse) [reply doc/newsboat.txt]: Now as I think of it again, you're absolutely right. The previous version definitely sounded quite unnatural and the preposition _of_ is correct. Totally missed that. I like your rewrite and going to use that.

File: doc/newsboat.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Minoru) [reply doc/newsboat.txt]: Yeah, on the second reading I can read it the way you do.

My understanding is that `gettext()` is commonly provided either by the C library or GNU gettext library. This is corroborated by [the history of GNU `gettext`](https://www.gnu.org/software/gettext/manual/html_node/History.html). An example of C library that didn't provide `gettext()` for some time is musl; [here is](https://gentoo.org/support/news-items/2014-10-22-upgrading-to-musl-1_1_5.html) a Gentoo post about upgrade to a version that adds `gettext()`.

So two readings are possible:
1. "GNU gettext is only needed if your libc doesn't provide `gettext()`"—link is outside the parens in this case;
2. "we depend on GNU gettext, and if it's not part of your libc, get it as a separate library here"—link is inside the parens.

Both readings are correct, I believe: you only need gettext if your libc doesn't provide it, *and* you can get it by that link. So just rewrite this item the way you rewrote the others, keeping the parenthesis, and we'll be done with this.

**Pull request merged**